sp800-63b/sec6_lifecycle.md

```diff
@@ -33,30 +33,21 @@ Enrollment and binding MAY be broken up into a number of separate physical encou
 
 #### 6.1.2. Post-Enrollment Binding
 
-Following enrollment, binding an additional authenticator to an account requires the use of an existing authenticator of the same type (or types). For example, binding a new single-factor OTP device requires the subscriber to authenticate with another *something you have* authentication factor. If the account has only one authentication factor bound to it (which is possible only at IAL 1/AAL 1), an additional authenticator of the same factor may be bound to it.
+Following enrollment, CSPs SHOULD permit the binding of additional authenticators to a subscriber's account. In order to do so, the CSP SHALL require the subscriber to first authenticate at the AAL of the authenticator to be added. When an authenticator is added, the CSP SHOULD send a notification to the subscriber.
 
-Binding an additional authenticator SHALL require the use of two different authentication factors, except as provided below.
-
-If the subscriber has only one of the two authentication factors, they SHALL repeat the identity proofing process, using the remaining authentication and SHOULD verify knowledge of some information collected during the proofing process to bind to the existing identity. In order to reestablish authentication factors at IAL 3, they SHALL verify the biometric collected during the proofing process.
-
-***Consider what proofing information the CSP is allowed to maintain. Privacy impact here?***
+If the account has only one authentication factor bound to it (which is possible only at IAL 1/AAL 1), an additional authenticator may be added, and if it is a different factor (e.g., *something you have* added when authenticated with *something you know*), the subscriber MAY request that the account be upgraded to AAL 2 (but still at IAL 1). Once this has been done, it SHALL no longer be possible for the subscriber to use single-factor authentication.
```

Comment:
we shift from 'authentication factor' to authenticator.  I think this will get confusing.  Perhaps, 'a single-factor authenticator'? And request seems iffy.  'the subscriber MAY be allowed to add the factor, therefore upgrading the account.'